build/onos-gui-build/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM  teracy/angular-cli
+
+COPY . /go/src/github.com/onosproject/onos-gui
```

Comment:
does docker need to be built every time the code changes? Could the folder be mounted with "-v" or something?